screens/AllVideosScreen.js

```diff
@@ -44,10 +43,8 @@ export default function AllVideosScreen() {
           auth: apiConfig.auth,
         })
         .then((data) => {
-          console.log(data.data.results);
           setData(data.data.results);
           setLoading(false);
-          console.log(loading);
         })
```

Comment:
I like these console logs being removed, but in general we should not check in changes that are unrelated to the ticket in question. It's fine here but let's avoid it moving forward.